--- config/condor-low-latency.init	2009-01-29 15:25:32.373671787 -0600
+++ config/condor-low-latency.init.el4	2009-01-29 15:22:40.612155292 -0600
@@ -36,7 +36,7 @@
 
 start() {
     echo -n $"Starting condor-low-latency daemon: "
-    daemon --pidfile $pidfile --check $prog $prog -p $pidfile --daemon
+    daemon --check $prog $prog -p $pidfile --daemon
     RETVAL=$?
     echo
     [ $RETVAL -eq 0 ] && touch $lockfile
@@ -45,7 +45,7 @@
 
 stop() {
     echo -n $"Stopping condor-low-latency daemon: "
-    killproc -p $pidfile $prog
+    killproc $prog
     RETVAL=$?
     echo
     [ $RETVAL -eq 0 ] && rm -f $lockfile
@@ -80,7 +80,7 @@
         fi
         ;;
     status)
-        status -p $pidfile $prog
+        status $prog
         RETVAL=$?
         ;;
     *)
